src/o365/spo/commands/list/list-view-list.spec.ts

```diff
@@ -0,0 +1,2260 @@
+import commands from '../../commands';
+import Command, { CommandValidate, CommandError, CommandOption } from '../../../../Command';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth, { Site } from '../../SpoAuth';
+const command: Command = require('./list-view-list');
+import * as assert from 'assert';
+import * as request from 'request-promise-native';
+import Utils from '../../../../Utils';
+
+describe(commands.LIST_VIEW_LIST, () => {
+  let vorpal: Vorpal;
+  let log: any[];
+  let cmdInstance: any;
+  let cmdInstanceLogSpy: sinon.SinonSpy;
+  let trackEvent: any;
+  let telemetry: any;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(auth, 'getAccessToken').callsFake(() => { return Promise.resolve('ABC'); });
```

Comment:
You will have to fake the this.getRequestDigestForSite method here like:
`sinon.stub(command as any, 'getRequestDigestForSite').callsFake(() => Promise.resolve({ FormDigestValue: 'ABC' }));`

This will prevent sending real https request and failure, but it will return just 'ABC' so the command can continue.